modules/tracktion_engine/timestretch/tracktion_RubberBand.cpp

```diff
@@ -0,0 +1,365 @@
+/*
+    ,--.                     ,--.     ,--.  ,--.
+  ,-'  '-.,--.--.,--,--.,---.|  |,-.,-'  '-.`--' ,---. ,--,--,      Copyright 2018
+  '-.  .-'|  .--' ,-.  | .--'|     /'-.  .-',--.| .-. ||      \   Tracktion Software
+    |  |  |  |  \ '-'  \ `--.|  \  \  |  |  |  |' '-' '|  ||  |       Corporation
+    `---' `--'   `--`--'`---'`--'`--' `---' `--' `---' `--''--'    www.tracktion.com
+
+    Tracktion Engine uses a GPL/commercial licence - see LICENCE.md for details.
+*/
+
+#include "tracktion_RubberBand.h"
+
+#ifdef __GNUC__
+ #pragma GCC diagnostic push
+ #pragma GCC diagnostic ignored "-Wsign-conversion"
+ #pragma GCC diagnostic ignored "-Wconversion"
+ #pragma GCC diagnostic ignored "-Wextra-semi"
+ #pragma GCC diagnostic ignored "-Wshadow"
+ #pragma GCC diagnostic ignored "-Wsign-compare"
+ #pragma GCC diagnostic ignored "-Wcast-align"
+ #if ! __clang__
+  #pragma GCC diagnostic ignored "-Wunused-but-set-variable"
+ #endif
+ #pragma GCC diagnostic ignored "-Wzero-as-null-pointer-constant"
+ #pragma GCC diagnostic ignored "-Wunused-variable"
+ #pragma GCC diagnostic ignored "-Wunused-parameter"
+ #pragma GCC diagnostic ignored "-Wpedantic"
+ #pragma GCC diagnostic ignored "-Wunknown-pragmas"
+#endif
+
+#define Point CarbonDummyPointName
+#define Component CarbonDummyCompName
+
+//#include "rubberband/rubberband-c.cpp"
+#include "../3rd_party/rubberband/src/StretcherProcess.cpp"
+#include "../3rd_party/rubberband/src/StretchCalculator.cpp"
+#include "../3rd_party/rubberband/src/StretcherChannelData.cpp"
+#include "../3rd_party/rubberband/src/StretcherImpl.cpp"
+#include "../3rd_party/rubberband/src/system/Allocators.cpp"
+#include "../3rd_party/rubberband/src/system/sysutils.cpp"
+#include "../3rd_party/rubberband/src/system/Thread.cpp"
+#include "../3rd_party/rubberband/src/system/VectorOpsComplex.cpp"
+// #include "rubberband/pommier/neon_mathfun.h"
+// #include "rubberband/pommier/sse_mathfun.h"
+#include "../3rd_party/rubberband/src/dsp/AudioCurveCalculator.cpp"
+#include "../3rd_party/rubberband/src/dsp/FFT.cpp"
+#include "../3rd_party/rubberband/src/dsp/Resampler.cpp"
+#include "../3rd_party/rubberband/src/base/Profiler.cpp"
+#include "../3rd_party/rubberband/src/audiocurves/CompoundAudioCurve.cpp"
+#include "../3rd_party/rubberband/src/audiocurves/ConstantAudioCurve.cpp"
+#include "../3rd_party/rubberband/src/audiocurves/HighFrequencyAudioCurve.cpp"
+#include "../3rd_party/rubberband/src/audiocurves/PercussiveAudioCurve.cpp"
+#include "../3rd_party/rubberband/src/audiocurves/SilentAudioCurve.cpp"
+#include "../3rd_party/rubberband/src/audiocurves/SpectralDifferenceAudioCurve.cpp"
+#include "../3rd_party/rubberband/src/speex/resample.c"
+// #include "rubberband/jni/Rubbersrc/BandStretcherJNI.cpp"
+
+#include "../3rd_party/rubberband/src/RubberBandStretcher.cpp"
+
+#undef Point
+#undef Component
+
+#ifdef __GNUC__
+ #pragma GCC diagnostic pop
+#endif
+
+bool RubberStretcher::setSpeedAndPitch(float speedRatio, float semitonesUp)
```

Comment:
There should be spaces before an opening parenthesis e.g.:
`setSpeedAndPitch (float speedRatio`
`float pitch = juce::jlimit (0.25f`